<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget_open"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_save"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu123">
    <property name="title">
     <string>文件操作</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选择当前引擎</string>
    </property>
    <addaction name="actionMicroSoftExcel"/>
    <addaction name="separator"/>
    <addaction name="actionThirdExcel"/>
    <addaction name="separator"/>
    <addaction name="actionWPSExcel"/>
   </widget>
   <addaction name="menu123"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMicroSoftExcel">
   <property name="text">
    <string>MSRWExcel</string>
   </property>
  </action>
  <action name="actionThirdExcel">
   <property name="text">
    <string>ThirdRWExcel</string>
   </property>
  </action>
  <action name="actionWPSExcel">
   <property name="text">
    <string>WPSRWExcel</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存Excel</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>打开Excel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
